src/commands/stats.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   },
   fn: async (msg) => {
```

Comment:
Top level async is redundant at this point
```suggestion
   fn: (msg) => {
```